docs/md/content/usage/tool_installation.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Installing Tools"
+description: "Learn how to install tools to PRM."
+category: narrative
+tags:
+  - tools
+  - usage
+  - install tool
+  - update tool
+  - list tools
```

Comment:
I'm not sure how kindly the systems will treat tags with spaces. Maybe best to use dashes?

```suggestion
  - install-tool
  - update-tool
  - list-tools
```